### PR TITLE
hwdb: quirk for kionix accelerometer on Asus TP500LB

### PR DESCRIPTION
**Pull request by M0Rf30**
This is a quirk for kionix accelerometer on Asus tp500lb. I've tested the following accel_mount_matrix setting

### CODE REVIEW

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #   /etc/udev/hwdb.d/61-sensor-local.hwdb
 # and add your rules there. To load the new rules execute (as root):
 #   systemd-hwdb update
-#   udevadm trigger `dirname $(udevadm info -n "/dev/iio:deviceXXX" -q path)`
+#   udevadm trigger -y `dirname $(udevadm info -n "/dev/iio:deviceXXX" -q path)`
```
Comment (hadess) [inline hwdb/60-sensor.hwdb:18]: This and...

File: hwdb/60-sensor.hwdb [hunk 2]
```diff
@@ -25,6 +25,9 @@
 # include your new rules, a description of the device, and the output of
 #   udevadm info --export-db
 #
+# For hwdb format and systemd behavior:
```
Comment (hadess) [inline hwdb/60-sensor.hwdb:28]: ... this, should be in a separate commit.

File: hwdb/60-sensor.hwdb [hunk 3]
```diff
@@ -34,12 +37,19 @@
 # eg. the identity matrix.
 #
 # [1]: https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=dfc57732ad38f93ae6232a3b4e64fd077383a0f1
-
+#
 #
 # Sort by brand, model
+#
+#########################################
+# AsusTek TP500LB
```
Comment (hadess) [inline hwdb/60-sensor.hwdb]: This should be the manufacturer name, not the model.

File: hwdb/60-sensor.hwdb [hunk 4]
```diff
@@ -34,12 +37,19 @@
...
+#########################################
+sensor:modalias:acpi:SMO8500*:dmi:*svn*ASUSTeK*:*pn*TP500LB*
+ ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 0
 
 #########################################
 # Winbook
 #########################################
 sensor:modalias:acpi:BMA250*:dmi:*svn*WinBook*:*pn*TW100*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 0
+
```
Comment (hadess) [inline hwdb/60-sensor.hwdb]: This isn't needed either

File: hwdb/60-sensor.hwdb (hunk 1 above)
Comment (keszybz) [reply hwdb/60-sensor.hwdb:18]: That's not needed, right?

Comment (keszybz) [reply hwdb/60-sensor.hwdb:18]: Oh, it seems it is. Never mind.

**Comment by keszybz** [comment]
@M0Rf30 can you adjust your PR following the comments from @hadess?

**Comment by keszybz** [comment]
Something strange happened here.
